docs/en/stock-management/movement.exit.md

```diff
@@ -2,29 +2,221 @@
+
+  <img src="./images/exit-select-patient.png" alt="Select Patient for Exit" width="600">
+
+   - Click in the small box labeled "Patient ID" on the left (indicated by the
+     blue arrow) to choose the type of patient search. By default, the system
+     lets you the search by patient ID. You can change the type of search, by
+     clicking in the box. The system offers 3 types of search (by ID / code,
+     by patient name or by barcode scanner). Select the type of search desired
+     by clicking on it.
+
+   - If a barcode reader is being used, scan the barcode now.  If not, enter
+     in the search box on the right, the patient's identifier or name,
+     depending on whether the choice was made for a search by ID or by
+     name. Enter the characters of the user's ID or name in the search field.
+     The system will suggest all matching names. Select the desired patient by
+     clicking on the desired name/ID.  It will then appear in the search box.
+
+   - Note that exits to patients now require an invoice.  Invoices here are
+     essentially prescriptions and are usually created by the medical staff
+     (using <i>menu</i> > Finance > Invoice).
+
+     If a barcode from an invoice is scanned, the patient ID and invoice will
+     be handled automatically.  If not, once a patient ID is entered, the form
+     will be populated with available invoices and the user can select the
+     correct only.
+
+When the patient selection form is complete, click on the blue **[Submit]**
+button. The Stock Exit form will be redisplayed with the name of the patient
+with blue highlighting.
+
+<img src="./images/exit-to-patient-form.png" alt="Exit to Patient Form - Populated" width="800">
+
+ - Then we follow the same process as before in terms of adding stock items to
+   be dispensed to the patient (stock exits).  Fill the form from top to
+   bottom.
+
+ - Change the date if necessary.
+
+ - In the "Description" field, complete the description as appropriate.
+
+ - Add stock items by clicking on the "Add" button.
+
+ - Complete filling out each line using the same process as in previous
+   descriptions of recording stock exits.
+
+ - Once the process is complete, press the blue "Submit" button on the bottom
+   left.
+
+ - A patient delivery slip will be displayed which can be printed.
+
+
+---
+
+## Stock Exit by Loss
+
+To record a stock exit due to loss, from the "Stock Exit" page, click on the
+big **[Loss]** button at the top of the form.  Since there is no depot or
+service to involved, it is just a loss and the system gives you direct access
+to the exit voucher to be completed. Here is what the form Loss page looks
+like.
+
+<img src="./images/exit-to-loss-form.png" alt="Exit to Loss Form" width="800">
+
+- Make sure that the correct Depot is selected.
+
+- To complete the form, follow the same logic as for stock exits to a depot or
+  service.
+
+
+[^1]: Stock exits to patients and services are _consumptions_ and influence
```

Comment:
Does this actually work?  This footnote?  I'm super curious to see! 😄

Reply:
I think it will.  I is github markdown.

Reply:
Just checked.  It works.  Look at the bottom of the Exit page.